tests/container_disk_test.go

```diff
@@ -219,13 +210,9 @@ var _ = Describe("[rfe_id:588][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 	Describe("[rfe_id:273][crit:medium][vendor:cnv-qe@redhat.com][level:component]Starting with virtio-win", func() {
 		Context("with virtio-win as secondary disk", func() {
 			It("[test_id:1467]should boot and have the virtio as sata CDROM", func() {
-				vmi := tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskAlpine))
+				vmi := libvmi.NewAlpine()
```

Comment:
Do we have an option for
`tests.AddEphemeralCdrom(vmi, "disk4", "sata", cd.ContainerDiskFor(cd.ContainerDiskVirtio))` ?

Reply:
I don't believe we do. It makes sense to add `libvmi.WithContainerImageCdrom`, but unless you insist, I would take this idea to another PR.

Reply:
>I would take this idea to another PR.
Absolutely! It just got my attention :+1: